src/clientmetricreport/ClientMetricReport.ts

```diff
@@ -530,6 +552,21 @@ export default class ClientMetricReport {
       media: MediaType.VIDEO,
       dir: Direction.UPSTREAM,
     },
+    videoUpstreamEncoderImplementation: {
```

Comment:
can you remind me if all these comes from getStats API?

Reply:
Yes, the metrics are all from `getStats`